components/webui/imports/api/search/server/publications.js

```diff
@@ -65,8 +68,12 @@ Meteor.publish(Meteor.settings.public.SearchResultsCollectionName, ({
             /* eslint-enable @stylistic/js/array-element-newline */
         ],
         limit: SEARCH_MAX_NUM_RESULTS,
+        // disable oplog should be always true since
+        // enable w/o a sort specifier is not supported
         disableOplog: true,
-        pollingIntervalMs: COLLECTION_POLL_INTERVAL_MILLIS,
+        pollingIntervalMs: (false === isExpectingUpdates) ?
+            COLLECTION_POLL_INTERVAL_MILLIS :
+            Infinity,
```

Comment:
```suggestion
        pollingIntervalMs: (true === isExpectingUpdates) ?
            COLLECTION_POLL_INTERVAL_MILLIS:
            Infinity,
```
It seems a bit counter intuitive that the CPU usage can be reduced with polling enabled. Are the latest changes verified?

Reply:
You've got me. The last verification was done for the version `isActivelyPolling`. then we changed to `false === isActivelyPolling`, and hence the error was generated and passed down.